api/Document.json

```diff
@@ -2767,13 +2767,14 @@
             "description": "<code>expandEntityReferences</code> parameter",
             "support": {
               "chrome": {
-                "version_added": "4"
+                "version_added": false
```

Comment:
[Per the comment](https://source.chromium.org/chromium/chromium/src/+/116ac6de3914734f7136874089a55a3922113f1d:third_party/WebKit/Source/core/dom/Document.idl;dlc=954f614cc71dbb9a8f5990600de09f7cf0cf53ad):

```
// Historically, this argument was never implemented and has been ignored.
```